### PR TITLE
style: remove blank line & white space formatting

### PR DESCRIPTION
**Pull request by yqrashawn**

### CODE REVIEW

**Comment by ChuhanJin** [comment]
Can one of the admins verify this patch?

**Comment by yqrashawn** [comment]
two-dot diff at https://github.com/scroll-tech/go-ethereum/pull/288/files
three-dot diff at https://github.com/scroll-tech/go-ethereum/pull/287/files

Please do check if conflicts resolved correctly. 
Files with conflicts can be found at https://github.com/scroll-tech/go-ethereum/pull/267

File: internal/ethapi/api.go [hunk 1]
```diff
@@ -835,11 +836,12 @@ func (s *PublicBlockChainAPI) GetStorageAt(ctx context.Context, address common.A
...
 type OverrideAccount struct {
-	Nonce     *hexutil.Uint64              `json:"nonce"`
-	Code      *hexutil.Bytes               `json:"code"`
-	Balance   **hexutil.Big                `json:"balance"`
-	State     *map[common.Hash]common.Hash `json:"state"`
-	StateDiff *map[common.Hash]common.Hash `json:"stateDiff"`
+	Nonce      *hexutil.Uint64              `json:"nonce"`
+	Code       *hexutil.Bytes               `json:"code"`
+	Balance    **hexutil.Big                `json:"balance"`
+	BalanceAdd **hexutil.Big                `json:"balanceAdd"`
```
Comment (0xmountaintop) [inline internal/ethapi/api.go]: image

why is `BalanceAdd` added?

**Comment by 0xmountaintop** [comment]
I think you forget to apply 

1. these in core/blockchain_test.go
image

2. this blank line in core/state/snapshot/generate.go
image


3. this blank line in eth/api_test.go
image


---

reference: https://github.com/scroll-tech/go-ethereum/compare/compare-pre-revert-formats..scroll-tech:go-ethereum:alpha?expand=1

**Pull request merged**